### ISSUE TITLE
Clueless Here

### ISSUE THREAD

**Issue by 7Moses** [roidrage/s3itch#14]
Hello, I am using Skitch 1.0.11, and I am trying to get it to work with s3itch.  My progress through set up ended at "git push heroku master," see https://www.box.com/shared/sjlq1wnkluame3srwl2u. Entering "git push heroku master," resulted in my terminal window returning the message: fatal: Not a git repository (or any of the parent directories): .git. I have likely just got the setup method wrong. Any insights appreciated, direction to guidance notes appreciated, thanks.

**Comment by roidrage** [comment]
Sorry, I think the installation instructions are missing a step `cd s3itch` before creating the Heroku app. If you `cd` into the s3itch directory, the rest should work a treat.

**Comment by 7Moses** [comment]
Thanks for the reply. Used cd s3itch, but take a look at this https://www.box.com/s/571wzise068s5idiaajg and let me know what I am doing wrong? Thanks

**Comment by roidrage** [comment]
You'll have to re-run the heroku create command in that directory.

**Comment by roidrage** [comment]
Isn't this the exact same comment and screencap you sent 10 hours ago?

**Comment by 7Moses** [comment]
Thanks I did and I used "S3 setup" on this page: https://devcenter.heroku.com/articles/s3#s3-setup, to enter my Amazon credentials. I am currently getting the "Object Not Found" error, see https://www.box.com/s/gtfh94jq6i2na9yh1cyx; I have checked the url is correct, http://<bucketname>.s3.amazonaws.com and I have also tried, http://<bucketname>.s3-website-us-east-1.amazonaws.com/. Any further insights appreciated, Best,

**Comment by roidrage** [comment]
Did you set up and configure a bucket for S3 and set the `S3_BUCKET` variable?

**Comment by 7Moses** [comment]
Yes I set up a bucket, the bucket name is deleted in my last post. Though I am unsure if i have the bucket configuration right, I will check google for guidance. Thanks, Best,
